cfgov/v1/models/molecules.py

```diff
@@ -31,6 +33,30 @@ class ImageText5050(blocks.StructBlock):
     link_url = blocks.URLBlock(required=False)
     link_text = blocks.CharBlock(max_length=100, required=False)
 
+    def clean(self, data):
+        tuples = dict(data.items())
```

Comment:
yerp good call. got rid of the title validation since its handled in the super and included a try catch so it handles the custom validation as well before returning to the frontend